ui/src/components/TaskDetailsCard/index.jsx

```diff
@@ -25,7 +25,7 @@ import Link from '../../utils/Link';
 
 @withStyles(theme => ({
   headline: {
-    paddingLeft: theme.spacing.triple,
+    paddingLeft: theme.spacing.double,
     paddingRight: theme.spacing.triple,
```

Comment:
Let's change this to `theme.spacing.double` for consistency.

Reply:
> Let's change this to `theme.spacing.double` for consistency.

Did you mean right-padding? I've committed with changes, please, review one more time.